platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/GenesisStateBuilder.java

```diff
@@ -47,7 +47,7 @@ private static PlatformState buildGenesisPlatformState(
 
         platformState.setCreationSoftwareVersion(appVersion);
         platformState.setRound(0);
-        platformState.setRunningEventHash(null);
+        platformState.setLegacyRunningEventHash(null);
```

Comment:
do you also need to set the new running hash here?

Reply:
Good idea, set this equal to the null hash. I was handling the situation where this field was null for a genesis state, but this is more elegant. Once we migrate mainnet, I can completely remove the logic that currently handles an initial state with a `null` value here.